nuget/AppCenter.nuspec

```diff
@@ -32,6 +32,12 @@
         <dependency id="System.Configuration.ConfigurationManager" version="4.6.0"/>
         <dependency id="Microsoft.Windows.SDK.Contracts" version="10.0.18362.2005"/>
       </group>
+      <group targetFramework="net5.0-windows7.0">
```

Comment:
Why is it called `net5.0-windows7.0`? Can we name it `net5.0-windows`.

Reply:
To avoid any issues during the packing nuget we need to set [the default version](https://github.com/NuGet/Home/issues/10897).